scripts/deploy.ts

```diff
@@ -98,8 +98,13 @@ export async function deploySuite(env: string = "", modules: string[] = [], crea
     const fermionSeaportWrapper = await FermionSeaportWrapper.deploy(...seaportWrapperConstructorArgs);
     const FermionFNFTPriceManager = await ethers.getContractFactory("FermionFNFTPriceManager");
     const fermionFNFTPriceManager = await FermionFNFTPriceManager.deploy();
+    const FermionFractionsERC20 = await ethers.getContractFactory("FermionFractionsERC20");
+    const fermionFractionsERC20 = await FermionFractionsERC20.deploy();
     const FermionFractionsMint = await ethers.getContractFactory("FermionFractionsMint");
-    const fermionFractionsMint = await FermionFractionsMint.deploy(bosonPriceDiscoveryAddress);
+    const fermionFractionsMint = await FermionFractionsMint.deploy(
+      bosonPriceDiscoveryAddress,
+      await fermionFractionsERC20.getAddress(),
+    );
     const FermionBuyoutAuction = await ethers.getContractFactory("FermionBuyoutAuction");
     const fermionBuyoutAuction = await FermionBuyoutAuction.deploy(bosonPriceDiscoveryAddress);
 
```

Comment:
Just a comment, if #398 will be merged before this, make sure to properly update  `deploymentComplete` statement for "FermionFractionsMint" that is coming in that PR.